lib/hexpm_web/router.ex

```diff
@@ -201,6 +204,13 @@ defmodule HexpmWeb.Router do
     get "/feeds/blog.xml", FeedsController, :blog
   end
 
+  scope "/login/web_auth", HexpmWeb do
+    pipe_through :api
+
+    post "/code", WebAuthController, :code
+    post "/access_key", WebAuthController, :access_key
```

Comment:
I would also probably implement a fallback controller for WebAuth, see https://hexdocs.pm/phoenix/Phoenix.Controller.html#action_fallback/1-examples

Reply:
Ah, I wasn't aware of fallback controllers. Thanks!